target/scripts/helper-functions.sh

```diff
@@ -288,27 +288,65 @@ export -f _monitored_files_checksums
 
 # ? --------------------------------------------- General
 
+# Outputs the DNS label count (delimited by `.`) for the given input string.
+# Useful for determining an FQDN like `mail.example.com` (3), vs `example.com` (2).
+function _get_label_length
+{
+  local INPUT=${1}
+  awk -F '.' '{ print NF }' <<< "${INPUT}"
```

Comment:
One-liner:
```suggestion
  awk -F '.' '{ print NF }' <<< "${1}"
```

Reply:
I usually prefer a named variable myself for clarity but in this case I guess it's not too important :sweat_smile: